src/utils/layout-helpers.ts

```diff
@@ -117,10 +120,26 @@ export function getPodModulePrefix(root: string): string | null {
   return podModulePrefix.trim().length > 0 ? podModulePrefix : null;
 }
 
+export function getRoot(fromDir: string) {
+  try {
+    const topLevel = childProcess.execSync('git rev-parse --show-toplevel', { encoding: 'utf8', cwd: fromDir });
```

Comment:
we can't afford any `sync` execution, because we don't have real file system (by default)

Reply:
reverted